decidim-comments/app/cells/decidim/comments/comment/show.erb

```diff
@@ -1,51 +1,89 @@
-<%= content_tag :div, id: "comment_#{model.id}", class: comment_classes, data: { comment_id: model.id } do %>
+<%= content_tag :div, id: "comment_#{model.id}", class: "comment relative", data: { comment_id: model.id } do %>
   <% if model.hidden? %>
     <%= render :moderation_data %>
   <% elsif model.deleted? %>
     <%= render :deletion_data %>
   <% else %>
     <div class="comment__header">
-      <div class="author-data">
-        <div class="author-data__main">
-          <%= render :author %>
-          <span>
-            <%= time_tag created_at, l(created_at, format: :decidim_short) %>
-          </span>
-          <% if edited? %>
-            <div class="author-data__extra">
-              <span class="comment__edited">
-                <%= t("decidim.components.comment.edited") %>
-              </span>
-            </div>
-          <% end %>
+      <span class="font-bold">
+        <%== cell("decidim/redesigned_author", author_presenter) %>
+      </span>
+      <span class="text-gray-2 text-sm">
+        <%= time_tag created_at, time_ago_in_words(created_at) %>
+      </span>
+      <% if edited? %>
+        <span class="label">
+          <%= t("decidim.components.comment.edited") %>
+        </span>
+      <% end %>
+
+      <div class="relative ml-auto">
+        <button id="dropdown-trigger-<%= context_menu_id %>" data-component="dropdown" data-target="dropdown-menu-<%= context_menu_id %>">
+          &#8230;
+        </button>
+
+        <div id="dropdown-menu-<%= context_menu_id %>" aria-hidden="true">
+          <ul class="dropdown dropdown__bottom divide-y divide-gray-3 px-4">
+            <% unless reloaded? %>
+              <li>
+                <%= cell("decidim/report_button", model, only_button: true, button_classes: "dropdown__item flex-row-reverse", modal_id: "flagModalComment#{model.id}") %>
+              </li>
+            <% end %>
+            <li>
+              <%= link_to "#{commentable_path("commentId" => model.id)}#comment_#{model.id}", target: "_blank", data: { "external-link": "false" }, class: "dropdown__item", title: t("decidim.components.comment.single_comment_link_title") do %>
+                <%= icon "share-line" %>
+                <span><%= t("decidim.components.comment.single_comment_link_title") %></span>
+              <% end %>
+            </li>
+            <% if model.authored_by?(current_user) %>
+              <li>
+                <button type="button" class="dropdown__item" data-dialog-open="<%= "editCommentModal#{model.id}" %>" title="<%= t("decidim.components.comment.edit") %>" aria-controls="<%= "editCommentModal#{model.id}" %>" aria-haspopup="dialog" tabindex="2">
+                  <%= icon "edit-line" %>
+                  <span><%= t("decidim.components.comment.edit") %></span>
+                </button>
+              </li>
+            <% end %>
+            <% if model.authored_by?(current_user) %>
```

Comment:
You could put the both actions (edit + delete) in a single `<% if model.authored_by?(current_user) %>` statement.

Reply:
Done!